config/manager/manager.yaml

```diff
@@ -19,8 +19,7 @@ spec:
       containers:
       - args:
         - "--metrics-addr=127.0.0.1:8080"
```

Comment:
Did you mean to remove `"--enable-leader-election"`?

Reply:
No, this was wrong, fixed.